pyanaconda/modules/storage/nvdimm/nvdimm.py

```diff
@@ -101,6 +101,11 @@ def get_devices_to_ignore(self):
         devices_to_ignore = set()
 
         for ns_name, ns_info in nvdimm.namespaces.items():
+            if ns_info.blockdev is None:
+                log.debug("%s will be skipped - NVDIMM namespace block device information "
```

Comment:
You could make this a little more elegant by ignoring all namespaces that are not in devdax or sector mode. You are already ignoring all non-sector mode namespaces in the `elif` below, so it would make the code a little bit more straightforward. (But that's just a nitpick, feel free to ignore me.)

Reply:
Good point, however, I'm thinking if I wouldn't prefer this solution with a comment to explain in case there would be other similar cases. It's just more robust. What do you think @poncovka ?

Reply:
I agree. We should still check `blockdev`. Please, add a new comment or update the commit  message.